### PR TITLE
add symstore recipe

### PR DESCRIPTION
**Pull request by elmirjagudin**
Adds recipe for 'symstore' pypi package.

Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by elmirjagudin** [comment]
for the record, I'm willing to be listed as maintainer for this recipe

File: recipes/symstore/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+test:
+  imports:
+    - symstore
+  commands:
+    - symstore --help
+
+about:
+  home: "https://github.com/elmirjagudin/symstore"
+  license: BSD
```
Comment (chrisburr) [inline recipes/symstore/meta.yaml]: Which BSD licence should this be? The most common is:
```suggestion
  license: BSD-3-Clause
  license_family: BSD
  license_file: LICENSE
```
Also they all require the text of the license to be included so you should add a copy to your GitHub repository. You can then tag a new version or include it in the recipe directory so conda can pick it up.

File: recipes/symstore/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  summary: "publish PDB and PE files to symbols store"
+  doc_url: "https://github.com/elmirjagudin/symstore/blob/v0.2.6/README.rst"
```
Comment (chrisburr) [inline recipes/symstore/meta.yaml]:
```suggestion
  doc_url: "https://github.com/elmirjagudin/symstore/blob/v{{ version }}/README.rst"
```

File: recipes/symstore/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+test:
+  imports:
+    - symstore
+  commands:
+    - symstore --help
+
+about:
+  home: "https://github.com/elmirjagudin/symstore"
+  license: BSD
+  summary: "publish PDB and PE files to symbols store"
```
Comment (chrisburr) [inline recipes/symstore/meta.yaml]:
```suggestion
  summary: "publish PDB and PE files to symbols store"
  dev_url: https://github.com/elmirjagudin/symstore/
```

File: recipes/symstore/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+{% set name = "symstore" %}
+{% set version = "0.2.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "f49ac8031b7572b5f3bcac3a8296d498fcd2a410225b3886f1c6b90b21c5bbf0"
```
Comment (marcelotrevisani) [inline recipes/symstore/meta.yaml]: Could you please remove the quotes from ``sha26``?
```suggestion
  sha256: f49ac8031b7572b5f3bcac3a8296d498fcd2a410225b3886f1c6b90b21c5bbf0
```

File: recipes/symstore/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - symstore=symstore.command_line:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/symstore/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/symstore/meta.yaml (hunk 1 above)
Comment (elmirjagudin) [reply recipes/symstore/meta.yaml]: I made a new release to pypi which contains the LICENSE file. I also decided to go with MIT license, after a closer review of license text :)

File: recipes/symstore/meta.yaml (hunk 2 above)
Comment (elmirjagudin) [reply recipes/symstore/meta.yaml]: updated to use version template

File: recipes/symstore/meta.yaml (hunk 3 above)
Comment (elmirjagudin) [reply recipes/symstore/meta.yaml]: added dev URL

File: recipes/symstore/meta.yaml (hunk 4 above)
Comment (elmirjagudin) [reply recipes/symstore/meta.yaml]: dropped the ""

File: recipes/symstore/meta.yaml (hunk 5 above)
Comment (elmirjagudin) [reply recipes/symstore/meta.yaml]: dropped the extra arguments

**Comment by elmirjagudin** [comment]
@conda-forge/help-python Will this package be automatically build and added to 'conda-forge' channel, now that it have been merged? Or do I need to to something more?

This 'symstore' package still does not appear on the 'conda-forge' channel:

```
$ conda install -c conda-forge symstore
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - symstore

Current channels:

  - https://conda.anaconda.org/conda-forge/linux-64
  - https://conda.anaconda.org/conda-forge/noarch
  - https://repo.anaconda.com/pkgs/main/linux-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/linux-64
  - https://repo.anaconda.com/pkgs/r/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.
```

**Comment by chrisburr** [comment]
Looks like it was affected by [this bug in staged-recipes](https://github.com/conda-forge/conda-smithy/issues/1154).

The quickest/easiest solution is to push a commit to the feedstock to trigger the CI.

**Comment by elmirjagudin** [comment]
Yepp, pushing an empty commit to the feedstock repo did the trick, now symstore package is ready and installable from conda-forge.

Thanks @chrisburr !

**Pull request merged**